tools/accuracy_checker/tests/test_whisper_evaluator.py

```diff
@@ -0,0 +1,88 @@
+"""
+Copyright (c) 2024 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from pathlib import Path
+from unittest.mock import MagicMock
+
+import pytest
+from accuracy_checker.evaluators.custom_evaluators.whisper_evaluator import (
+    GenAIWhisperPipeline, OptimumWhisperPipeline, HFWhisperPipeline,
+    WhisperEvaluator, normalize_transcription)
+from datasets import load_dataset
+from optimum.exporters.openvino.convert import export_tokenizer
+from optimum.intel.openvino import OVModelForSpeechSeq2Seq
+from transformers import AutoTokenizer,AutoProcessor
+
+
+def export_model(model_id, output_dir):
+    tokenizer = AutoTokenizer.from_pretrained(model_id)
+    processor = AutoProcessor.from_pretrained(model_id)
+    base_model = OVModelForSpeechSeq2Seq.from_pretrained(model_id)
+
+    base_model.save_pretrained(output_dir)
+    tokenizer.save_pretrained(output_dir)
+    processor.save_pretrained(output_dir)
+    export_tokenizer(tokenizer, output_dir)
+
+model_name = "openai/whisper-tiny"
+model_dir = Path("/tmp/whisper-tiny")
+
+# Export the model
+export_model(model_name, model_dir)
+
+# Load a single sample from the dataset
+dataset = load_dataset("openslr/librispeech_asr", "clean", split="validation", streaming=True, trust_remote_code=True)
+sample = next(iter(dataset))
+ground_truth = sample["text"]
+input_data = [sample["audio"]["array"]]
+input_meta = [{"sample_rate": sample["audio"]["sampling_rate"]}]
+identifiers = [sample["id"]]
+# print(ground_truth)
```

Comment:
Please clean up the code a bit and remove print. Also, you need to remove the directory after test suite finishes. You can define `teardown_module()` function for that.

Reply:
also looks like you need to install datasets in test requirements
```
 tools/accuracy_checker/tests/test_whisper_evaluator.py:23: in <module>
    from datasets import load_dataset
E   ModuleNotFoundError: No module named 'datasets'
```

Reply:
@pwolnows I believe you have enough permissions to open github actions status, right? still some dependencies missed
https://github.com/openvinotoolkit/open_model_zoo/actions/runs/12442951517/job/34781160595?pr=3990